proto/stride/stakeibc/trade_route.proto

```diff
@@ -39,15 +41,32 @@ message TradeRoute {
   // Currently Osmosis is the only trade chain so this is an osmosis pool id
   uint64 pool_id = 9;
 
-  // Spot price is a decimal ratio of the input to output denom as a string
-  string spot_price = 10;
+  // price of the output denom relative to the input denom
+  // this is update periodically via an ICQ
+  string host_token_price = 10 [
```

Comment:
I don't love this name because it isn't clear what the host price is being priced against... or that it is the ibc denoms which will be involved etc.  Maybe a clearer name for what we are really doing would be would be recent_swap_ratio or something -- this also implies that it could be out of date because it makes a reader ask, how recent? But I think that is a good thing and just more self contained as a name

Reply:
went with swap_price - pushed in latest commit